packages/web-components/src/search/product-search-overlay.ts

```diff
@@ -158,15 +170,17 @@ export class ProductSearchOverlay extends LitElement {
                 searchResult.product = result;
                 return searchResult;
             }) ?? [];
-
+            this.redirects = productSearchResult.redirects;
+            const redirects: SearchResult[] = productSearchResult.redirects?.filter(x => x.data?.Title && URL.canParse(x.destination ?? '')).map(x => ({ redirect: x })) ?? [];
```

Comment:
Can't we do something smarter than it having to be called "Title"?

It might be a bit overkill to do template overwriting just for this, but I spent 15 minutes debugging why it didn't work for Kop & Kande. 

Maybe an optional property that tells it what key to look for? 🤷‍♂️